tests/e2e/config/wordpress-debug-log/log-ignore-list.js

```diff
@@ -16,7 +16,9 @@ export const logIgnoreList = {
 		'PHP Notice:  Trying to access array offset on value of type null in /var/www/html/wp-includes/rest-api/class-wp-rest-request.php',
 		'PHP Notice:  Trying to access array offset on value of type bool in /var/www/html/wp-includes/theme.php',
 	],
-	nightly: [
+	// Any entries that should be ignored in all versions.
+	ALL: [
+		'tail: /var/www/html/wp-content/debug.log has appeared',
 		// Can be removed once WordPress AMP Plugin removes the deprecated function call.
 		// See: https://github.com/ampproject/amp-wp/issues/7619
 		'PHP Deprecated:  Function _admin_bar_bump_cb is deprecated since version 6.4.0! Use wp_enqueue_admin_bar_bump_styles instead. in /var/www/html/wp-includes/functions.php',
```

Comment:
This came up again and has been fixed here 
<img width="1440" alt="image" src="https://github.com/user-attachments/assets/89e07636-a217-4b7b-a507-0c94ec081a81" />